ansible_collections/arista/avd/roles/eos_validate_state/anta_integration.md

```diff
@@ -148,108 +139,106 @@ title: Ansible Collection Role eos_validate_state - Preview Integration with ANT
 
-# Fabric name used in the reports name
-fabric_name: "all"
+# Output for test results JSON files if save_results is set to true.
+test_results_dir_name: <str; | default="test_results">
+test_results_dir: <str; | default="{{ eos_validate_state_dir }}/{{ test_results_dir_name }}">
 
-# Reports name
-eos_validate_state_md_report_path: "{{ eos_validate_state_dir }}/{{ fabric_name }}-state.md"
-eos_validate_state_csv_report_path: "{{ eos_validate_state_dir }}/{{ fabric_name }}-state.csv"
+# Fabric name used in the reports name.
+fabric_name: <str; | default="all">
 
-# Input directory for custom ANTA catalogs:
-custom_anta_catalogs_dir_name: "custom_anta_catalogs"
-custom_anta_catalogs_dir: "{{ root_dir }}/{{ custom_anta_catalogs_dir_name }}"
+# Reports name.
+eos_validate_state_md_report_path: <str; | default="{{ eos_validate_state_dir }}/{{ fabric_name }}-state.md">
+eos_validate_state_csv_report_path: <str; | default="{{ eos_validate_state_dir }}/{{ fabric_name }}-state.csv">
 
-# Allow different manufacturers
-accepted_xcvr_manufacturers: "{{ validation_role.xcvr_own_manufacturers | arista.avd.default(['Arastra, Inc.', 'Arista Networks']) }}"
+# Input directory for custom ANTA catalogs.
+custom_anta_catalogs_dir_name: <str; | default="custom_anta_catalogs">
+custom_anta_catalogs_dir: <str; | default="{{ root_dir }}/{{ custom_anta_catalogs_dir_name }}">
 
-# Allow different states for power supplies
-accepted_pwr_supply_states: "{{ validation_role.pwr_supply_states | arista.avd.default(['ok']) }}"
+# Allow different manufacturers.
+accepted_xcvr_manufacturers: <list; | default="['Arastra, Inc.', 'Arista Networks']">
 
-# Allow different states for fans
-accepted_fan_states: "{{ validation_role.fan_states | arista.avd.default(['ok']) }}"
+# Allow different states for power supplies.
+accepted_pwr_supply_states: <list; default=['ok']>
 
-# Generate CSV results file
-validation_report_csv: "{{ validation_role.validation_report_csv | arista.avd.default(true) }}"
+# Allow different states for fans.
+accepted_fan_states: <list; default=['ok']">
 
-# Generate MD results file
-validation_report_md: "{{ validation_role.validation_report_md | arista.avd.default(true) }}"
+# Generate CSV results file.
+validation_report_csv: <bool; default=True>
 
-# Print only FAILED tests
-only_failed_tests: "{{ validation_role.only_failed_tests | arista.avd.default(false) }}"
+# Generate MD results file.
+validation_report_md: <bool; default=True>
 
+# Print only FAILED tests.
+only_failed_tests: <bool; default=False>
 
-# Variable to enable ANTA eos_validate_state
-# Defaults to false as ANTA is currently in preview
-use_anta: false
+# Variable to enable ANTA eos_validate_state.
+use_anta: <bool; default=False>
 
-# Save each device test catalog to 'test_catalogs_dir'. Defaults to false.
-save_catalog: false
+# Save each device test catalog to 'test_catalogs_dir'.
+save_catalog: <bool; default=False>
```

Comment:
Should it not be lower case given that it is YAML?
```suggestion
save_catalog: <bool; default=false>
```

Reply:
We use capitalized True/False in our other roles: 
![image](https://github.com/aristanetworks/avd/assets/20460062/fdb0dba9-4974-4df0-9479-7a80b20b7423)

Reply:
I submit to this law of thats how we have been doing it before. :)